### DISCUSSION TITLE
Custom Component Margin

### DISCUSSION THREAD

**Discussion by Kuuchuu** [Budibase/budibase#4331] [Help] [answered]
Heyo! Is there any way to set a component to use a custom margin? The options go from 20px to 32px, when really I need something between there, such as 24px for example.
Any way to accomplish this? Or maybe a way to type in a custom margin could be added?
Thanks!

**Comment by mjashanks** [comment, marked as answer]
Yes, you can use the `Edit custom CSS` button:

![image](https://user-images.githubusercontent.com/3524181/152610762-0762c9fe-f1a8-45ac-bcac-053466bb386c.png)
